lib/ui/form/widget/form_survey_answer_multi_choice.dart

```diff
@@ -0,0 +1,121 @@
+
+class FormSurveyAnswerMultiChoice extends StatefulWidget {
+  final QuestionModel question;
+
+  const FormSurveyAnswerMultiChoice({super.key, required this.question});
+
+  @override
+  State<FormSurveyAnswerMultiChoice> createState() =>
+      _FormSurveyAnswerMultiChoiceState();
+}
+
+class _FormSurveyAnswerMultiChoiceState
+    extends State<FormSurveyAnswerMultiChoice> {
+  final List<int> _selectedIndexes = [];
+  late List<String> _options = [];
+
+  @override
+  void initState() {
+    super.initState();
+    _options = widget.question.answers.map((element) => element.text).toList();
+  }
+
+  Widget _buildListItem(String title, bool isSelected) {
+    final selectedTextStyle = Theme.of(context).textTheme.labelMedium;
+    final unSelectedTextStyle = Theme.of(context)
+        .textTheme
+        .labelMedium!
+        .copyWith(fontWeight: FontWeight.w400);
+    final textStyle = isSelected ? selectedTextStyle : unSelectedTextStyle;
+    return SizedBox(
+      width: MediaQuery.of(context).size.width -
+          AppDimensions.answerMultiChoiceMarginLength,
+      height: AppDimensions.answerMultiChoiceItemHeight,
+      child: Row(
+        children: [
+          Expanded(
+            child: Text(
+              title,
+              style: textStyle,
+              maxLines: 1,
+              overflow: TextOverflow.ellipsis,
+            ),
+          ),
+          Container(
+            width: AppDimensions.answerMultiChoiceCircleSize,
+            height: AppDimensions.answerMultiChoiceCircleSize,
+            decoration: BoxDecoration(
+              shape: BoxShape.circle,
+              border: Border.all(
+                width: AppDimensions.answerMultiChoiceCircleBorderWidth,
+                color: Colors.white,
+              ),
+            ),
+            child: isSelected
+                ? Container(
+                    width: AppDimensions.answerMultiChoiceCircleSize,
+                    height: AppDimensions.answerMultiChoiceCircleSize,
+                    decoration: const BoxDecoration(
+                      shape: BoxShape.circle,
+                      color: Colors.white,
+                    ),
+                    child: const FittedBox(
+                      child: Icon(
+                        Icons.check,
+                        size: AppDimensions.answerMultiChoiceCircleSize,
+                        color: Colors.black,
+                      ),
+                    ),
+                  )
+                : Container(),
+          )
+        ],
+      ),
+    );
+  }
+
+  Widget _buildItems(String title, bool isSelected, int index) {
+    return GestureDetector(
+      onTap: () {
+        setState(() {
+          if (_selectedIndexes.contains(index)) {
+            _selectedIndexes.remove(index);
+          } else {
+            _selectedIndexes.add(index);
+            _selectedIndexes.sort();
+          }
+        });
+      },
+      child: _buildListItem(title, isSelected),
+    );
+  }
```

Comment:
`_buildItems` (in plural) does not make sense as it builds a single item here. We can combine it into `_buildListItem` or find a better name.